File: VB/MvcDashboardDataSources/Configuration/EFDataSourceConfigurator.vb
Imports DevExpress.DashboardCommon
Imports DevExpress.DashboardWeb
Imports DevExpress.DataAccess.EntityFramework

Namespace MvcDashboardDataSources.Configuration
	Public Class EFDataSourceConfigurator
		Public Shared Sub ConfigureDataSource(ByVal storage As DataSourceInMemoryStorage)
			' Registers an Entity Framework data source.
			Dim efDataSource As New DashboardEFDataSource("EF Data Source")
			efDataSource.ConnectionParameters = New EFConnectionParameters(GetType(OrderContext))
			storage.RegisterDataSource("efDataSource", efDataSource.SaveToXml())
		End Sub
	End Class
End Namespace

File: CS/MvcDashboardDataSources/Configuration/EFDataSourceConfigurator.cs
using DevExpress.DashboardCommon;
using DevExpress.DashboardWeb;
using DevExpress.DataAccess.EntityFramework;

namespace MvcDashboardDataSources.Configuration {
    public class EFDataSourceConfigurator {
        public static void ConfigureDataSource(DataSourceInMemoryStorage storage) {
            // Registers an Entity Framework data source.
            DashboardEFDataSource efDataSource = new DashboardEFDataSource("EF Data Source");
            efDataSource.ConnectionParameters = new EFConnectionParameters(typeof(OrderContext));
            storage.RegisterDataSource("efDataSource", efDataSource.SaveToXml());
        }
    }
}